native/cocos/platform/android/java/src/com/cocos/lib/CocosHelper.java

```diff
@@ -308,10 +308,12 @@ public static long[] getObbAssetFileDescriptor(final String path) {
         return array;
     }
 
+    private static Display displayCache = null;
```

Comment:
Why need cache? Is it performance consuming?

Reply:
Yes

Reply:
Any performance data shared?

Reply:
Not yet. Recover it if necessary.

Reply:
As it make codes not clean. If not any result shows it will improve performance, then it is better not modify it. And it is not invoked frequently.

Reply:
Recovered.